frontend/src/js/components/tenants/tenantlist.tsx

```diff
@@ -0,0 +1,177 @@
+// Copyright 2024 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import React, { useCallback, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { ArrowForward as ArrowForwardIcon, Check as CheckIcon } from '@mui/icons-material';
+
+import { getDeviceLimit } from '@northern.tech/store/devicesSlice/selectors';
+import { getTenantsList } from '@northern.tech/store/organizationSlice/selectors';
```

Comment:
```suggestion
import { getDeviceLimit, getTenantsList } from '@northern.tech/store/selectors';
```
they should be all exported there too